### ISSUE TITLE
Add references to Citations section of Report

### ISSUE THREAD

**Issue by DEQrmichie** [DEQcdonald/st_report_files#17]
Need to add the following references to Citations section of the Rmarkdown report.

Bear Creek Watershed TMDL
Clear Lake TMDL
Garrison Lake TMDL
Malheur River Basin TMDL and WQMP
Snake River Hells Canyon TMDL
Tenmile Lakes TMDL
Tualatin Subbasin TMDL and WQMP
Umatilla River Basin TMDL and WQMP
Umpqua Basin TMDL and WQMP
Upper Klamath and Lost River Subbasin Nutrient TMDL and WQMP
Upper Klamath Lake Drainage TMDL

Oregon Department of Environmental Quality [DEQ]. 2020. Methodology for Oregon’s 2018 Water Quality Report and List of Water Quality Limited Waters. https://www.oregon.gov/deq/wq/Documents/irMethodologyF1820.pdf

**Comment by DEQygrund** [comment]
The citations in Ryan's list have been added to the Rmarkdown report.

Please note that - 

- Clear Lake TMDL: the year in the tmdl database (1992) is different from that on the website (1991). Need to verify.

- Umatilla River Basin TMDL and WQMP: the year in the tmdl database (2000) is different from that in the tmdl document (2001). 2001 is used in the Report. Need to update tmdl database.

In addition, below documents that were in the Citations section have been removed because they are not used in the 2020 Report.

Oregon Department of Environmental Quality [DEQ]. 1992. TMDLs for the Yamhill River. https://www.oregon.gov/deq/FilterDocs/YamhillTMDL1992.pdf

Oregon Department of Environmental Quality [DEQ]. 1998. Columbia Slough TMDL. https://www.oregon.gov/deq/FilterDocs/columbiasloughtmdl.pdf

**Comment by DEQrmichie** [comment]
Thanks! I checked and for Clear Lake, DEQ issued the TMDL in 1991 and EPA approved it in 1992. I think that is the reason for the discrepancy. Therefore I think Clear Lake should be referenced for 1991. You are correct that the Umatilla should be 2001. I corrected these in the TMDL database and also a few others I found. The changes were committed in https://github.com/DEQcdonald/odeqtmdl/commit/627c0e824d564e6a9a6eec81bfe9ac8d448d06b7.

Also note I added a new table of tmdl actions `odeqtmdl::tmdl_actions()`. This brings up a summary dataframe of all the TMDL actions including issue year, issue agency, etc.
